yarn-project/aztec-nr/compressed-string/src/compressed_string.nr

```diff
@@ -0,0 +1,139 @@
+use dep::aztec::types::type_serialization::TypeSerializationInterface;
+use dep::protocol_types::utils::field::field_from_bytes;
+use dep::std;
+
+// A Fixedsize Compressed String.
+// Essentially a special version of Compressed String for practical use.
+struct FixedsizeCompressedString{
```

Comment:
I think calling it a FieldCompressedString would be a bit clearer as it would tell you how much info can be put into it.